rc/core/c-family.kak

```diff
@@ -3,7 +3,7 @@ hook global BufCreate .*\.(cc|cpp|cxx|C|hh|hpp|hxx|H)$ %{
 }
 
 hook global BufSetOption filetype=c\+\+ %{
-    set-option buffer filetype cpp
+    hook -once buffer NormalIdle '' "set-option buffer filetype cpp"
```

Comment:
I changed this hook because the previous was prone to weird/racy behavior, with the filetype=cpp hook getting triggered twice.